src/omnicore/doc/release-notes.md

```diff
@@ -1 +1,236 @@
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.3.md
+
+Headers-first synchronization
+-----------------------------
+
+With the upgrade to Bitcoin Core 0.10 headers-first synchronization is supported and blocks are downloaded in parallel.
+
+Blocks are no longer stored in order on disk, and as result, the block files and databases are not backwards-compatible with older versions of Bitcoin Core prior 0.10.
+
+Dust threshold values
+---------------------
+
+The default `"minrelaytxfee"` was raised from `0.00001 BTC` to `0.00005 BTC`in Bitcoin Core 0.10.3 as temporary measure against massive memory pool bloat.
+
+This has an impact on the output values of Omni transactions, which are chosen to be as low as possible. As per default, Omni transactions created with Master Core had output values between `0.00000546 BTC` and `0.00000882 BTC`, whereby the new output values are between `0.0000273 BTC` and `0.0000441 BTC`.
+
+To continue to create transactions with lower values, start Omni Core with `-minrelaytxfee=0.00001` or add the following to your `bitcoin.conf`:
+```
+minrelaytxfee=0.00001
+```
+
+A lower `"minrelaytxfee"` may result in higher memory consumption, but is usually safe to use.
+
+Rebranding to Omni Core
+-----------------------
+
+Master Core was re-branded to Omni Core on all levels:
+
+- `mastercored`, `mastercore-cli` and `mastercore-qt` were renamed to `omnicored`, `omnicore-cli` and `omnicore-qt`
+- the debug log file `mastercore.log` was renamed  to `omnicore.log`
+- the hardcoded token SP#1 was renamed from `"MasterCoin"` to `"Omni"`
+- the hardcoded token SP#2 was renamed from `"Test MasterCoin"` to `"Test Omni"`
+- the user interface now refers to `"Omni"`, `"OMNI"`, `"Test Omni"` or `"TOMNI"` instead of `"Mastercoin"`, `"MSC"`, `"Test Mastercoin"` or `"TMSC"`
+
+Incompatible API changes
+------------------------
+
+The field `"subaction"` in the RPC response for traditional DEx orders of `"gettransaction_MP"` as well as `"listtransaction_MP"` was renamed to `"action"`.
+
+The values associated with `"action"` were renamed from `"New"`, `"Update"`, `"Cancel"` to `"new"`, `"update"`, `"cancel"`.
+
+Notable changes
+===============
+
+Feature activations
+-------------------
+
+TODO
+
+Data embedding with OP_RETURN
+-----------------------------
+
+TODO
+
+Distributed token exchange
+--------------------------
+
+TODO
+
+JSON-RPC API updates
+--------------------
+
+The JSON-RPC API was extended significantly, and now supports sending transactions of any transaction type, and the creation of raw Omni transactions.
+
+Several new API calls are available to retrieve information about unconfirmed or unconfirmed Omni transactions, and the state of the system.
+
+Please see the JSON-RPC API documentation for more details:
+
+- https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md
+
+Debug logging categories
```

Comment:
Debug logging could do with a cleanup actually - but for the time being I've run through and put together the following - do you think we should remove the 'currently unused' entries?

Let me know if I've missed anything...

**Debug logging categories**

Debug logging is categorized and default logging options may be changed with the ```-omnidebug``` parameter.  Use multiple ```-omnidebug``` parameters to specify multiple categories.

The following categories are available:

```parser``` : Log additional information when parsing
```parser_data``` : Log verbose additional information when parsing
```parser_dex``` : Log additional information about DEx payments when parsing
```parser_readonly``` : Log extra information when parsing in read only mode (eg via RPC interface)
```verbose``` : Log verbose information about sender identification
```verbose2``` : Currently unused
```verbose3``` : Currently unused
```vin``` : Log additional information about the inputs used when parsing
```script``` : Log additional information about the scripts used when parsing
```dex``` : Log additional information about DEx transactions
```send``` : Currently unused
```tokens``` : Log additional information about input selection
```spec``` : Warn about non-sequential sequence numbers in Class B transactions
```exo``` : Log additional information about actions involving the Exodus address
```tally``` : Log before/after values when balances are updated
```sp``` : Log additional information about actions involving smart property updates
```sto``` : Log additional information about Send To Owners calculations
```txdb``` : Log additional information about interactions with the transaction database
```tradedb``` : Log additional information about interactions with the trades database
```persistence``` : Log additional information about interactions with the persistent state files
```ui``` : Currently unused
```pending``` : Log additional information about pending transactions
```metadex1``` : Log additional information about MetaDEx transactions
```metadex2``` : Log the state of the MetaDEx before and after each MetaDEx transaction
```metadex3``` : Log the state of the MetaDEx before and after each MetaDEx action and log each MetaDEx object during iteration of MetaDEx maps
```packets``` : Log additional information about the payload included within each Omni transaction
```packets_readonly``` : Log additional information about the payload included within each Omni transaction when parsing in read only mode (eg via RPC interface)
```walletcache``` : Log additional information about the wallet cache and hits/misses
```consensus_hash``` : Log additional information about each data point added to the consensus hash
```consensus_hash_every_block``` : Generate and log a consensus hash for every block processed
```alerts``` : Log additional information about alert tranasctions
```none``` : Disable all logging categories
```all``` : Enable all logging categories (note, extremely verbose)

Examples: 
```
-omnidebug=all
```
```
-omnidebug=parser -omnidebug=parser_data -omnidebug=vin -omnidebug=script
```